docs/sql/statements/update.rst

```diff
@@ -53,3 +59,13 @@ Parameters
 :condition:
   An expression that returns a value of type boolean. Only rows for
   which this expression returns true will be updated.
+
+:output_expression:
+
+	An expression to be computed and returned by the UPDATE command after each
+	row is updated. The expression can use any column names of the table or * to
+	return all columns.
+
+:output_name:
+
+	A name to use for the result of an output expression.
```

Comment:
`returning * as foo` is not supported by pg, is it worth mentioning ?

Reply:
I think it should be clear from the synposis.